orc8r/cloud/go/services/certifier/obsidian/handlers/handlers.go

```diff
@@ -0,0 +1,208 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/labstack/echo"
+	"golang.org/x/crypto/bcrypt"
+
+	"magma/orc8r/cloud/go/obsidian"
+	"magma/orc8r/cloud/go/services/certifier"
+	certProtos "magma/orc8r/cloud/go/services/certifier/protos"
+	"magma/orc8r/cloud/go/services/certifier/storage"
+)
+
+const (
+	UserParam  = ":username"
+	User       = "user"
+	ListUser   = obsidian.V1Root + User
+	ManageUser = ListUser + obsidian.UrlSep + UserParam
+	Login      = ListUser + obsidian.UrlSep + "login"
+)
+
+func GetHandlers(storage storage.CertifierStorage) []obsidian.Handler {
+	ret := []obsidian.Handler{
+		{Path: ListUser, Methods: obsidian.GET, HandlerFunc: getListUserHandler(storage)},
+		{Path: ListUser, Methods: obsidian.POST, HandlerFunc: getCreateUserHandler(storage)},
+		{Path: ManageUser, Methods: obsidian.PUT, HandlerFunc: getUpdateUserHandler(storage)},
+		{Path: ManageUser, Methods: obsidian.DELETE, HandlerFunc: getDeleteUserHandler(storage)},
+		{Path: Login, Methods: obsidian.POST, HandlerFunc: getLoginHandler(storage)},
+	}
+	return ret
+}
+
+func getListUserHandler(storage storage.CertifierStorage) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		users, err := storage.ListUser()
+		if err != nil {
+			return echo.NewHTTPError(http.StatusInternalServerError, err)
+		}
+		return c.JSON(http.StatusOK, users)
+	}
+}
+
+type CreateUserRequest struct {
+	User struct {
+		Username string `json:"username"`
+		Password string `json:"password"`
+	} `json:"user"`
+	Policy struct {
+		Effect    string   `json:"effect"`
+		Action    string   `json:"action"`
+		Resources []string `json:"resource"`
+	} `json:"policy"`
+}
```

Comment:
This will work great, but for uniformity let's define this via swagger object, and then use that generated swagger object in the handlers